app/javascript/components/froggo-team-form.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <span>Nombre del equipo:</span><br><br>
+    <input type="text" v-model="team_name"><br><br>
+    <span>Agregar miembro:</span><br><br>
+    <users-dropdown
+        :users="users"
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 6 spaces but found 8 spaces                               vue/html-indent